fiboa_cli/datasets/fs_de_bb.py

```diff
@@ -0,0 +1,144 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# Local URI added to the repository for initial conversion, Original Source https://beta.source.coop/esa/fusion-competition/
+URI = "/home/byteboogie/fieldscapes/germany/fs_de_bb.gpkg"
```

Comment:
I'm not sure how to proceed with this. It's not really useful for the general public if there's no way to get the source data. Any thoughts from the fieldscapes members?

Reply:
Chrish asked to keep the local folder for now later we can change this to maybe Source URI? Not sure need extensive discussion on this.